### DISCUSSION TITLE
Change the model color on the plate??

### DISCUSSION THREAD

**Discussion by gasprints** [OrcaSlicer/OrcaSlicer#521] [Q&A] [answered]
How can I change the color on the plate?  This black is so terrible and doesn't show features or details at all.  I've tried changing the "default" color from the filament settings window and nothing happens when I change the color.  Even after restart.  When searching the forum the one closed discussion found a way that, as described, isn't in my version of orca slicer (latest) and seems the default at the time was white.
<img width="725" alt="Screenshot 2023-03-18 at 12 27 47 PM" src="https://user-images.githubusercontent.com/128255447/226132978-8535458c-b731-4f79-897a-114af3e2cf97.png">

**Comment by Manfredo1952** [comment]
You have an AMS attached ?  Can't you just change the colour of the object to match the colour of the filament ? 
![image](https://user-images.githubusercontent.com/56330568/226153022-6ed5a7bd-cb12-48e2-a971-b5fb5ce7b5f3.png)

**Comment by Alex00101** [comment]
I second that question. I don't have a multi color printing setup (Voron profile) but despite setting filament color appropriately, the model is always black on the plate and thumbnail....

**Reply by PuterPro** [reply]
Not sure what you guys are experiencing. I have OrcaSlicer 1.5, set to Dark Mode, and it shows filament color fine on Prepare screen, Red for Red, etc. You might open the filament editor and change the color, save, then change it back to green as you showed. Also might remove and reinstall OrcaSlicer. That does _not_ lose your settings, they're stored on the Bambu Cloud. That will cause it to reload the profiles you made.

**Reply by Alex00101** [reply, marked as answer]
OK, I'm in a dark mode (v1.5) on Windows 11.  Prepare Screen looks like this.
![ZScreenshot_1jpg](https://user-images.githubusercontent.com/27343158/227754948-0108bf7e-aec7-450d-865b-3c653e00fe1f.jpg)
Filament Color is set like this:
![ZScreenshot_2jpg](https://user-images.githubusercontent.com/27343158/227754988-792d2072-9369-457b-bd44-a438b5c11f05.jpg)
That still displays model in black.
However when I click this square
![ZScreenshot_3](https://user-images.githubusercontent.com/27343158/227755039-23fb5f2a-68b4-4da2-93bc-52e128c9b1f6.jpg)
I can set up the color it is going to be displayed on the plate (yet to check the thumbnails):
![ZScreenshot_4](https://user-images.githubusercontent.com/27343158/227755065-85a8e636-4283-47d1-8bad-5dda0a13b707.jpg)
So I guess this is works just somewhat different from other slicers I used to,

**Reply by gasprints** [reply]
Thanks!  I actually found this out the day after posting, the black box with the "1" was just black on my screen due to being black on black.  I didn't notice it at all and clearly my monitor displays blacks deeper than other screens.  Definitely different than other slicers, I've been using S3D for so long even cura looks foreign now
